lib/src/models/twitch_create_clip.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'twitch_create_clip.freezed.dart';
+part 'twitch_create_clip.g.dart';
+
+@freezed
+class TwitchCreateClip with _$TwitchCreateClip {
+  const factory TwitchCreateClip({
+    @JsonKey(name: 'edit_url') required String editUrl,
+    @JsonKey(name: 'id') required String id,
```

Comment:
`@JsonKey` is not needed:
```suggestion
    required String id,
```